practices/risk_analysis.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: risk-analysis
+stages:
+  - development
+short_description: Risk Analysis and Management is a key project management practice to ensure that the least number of surprises occur while the project is underway
+tags:
+  - risk analysis
+  - software documentation
+keywords:
+  - risk analysis
+  - documentation
+  - task
+---
+
+# Risk Analysis
+
+**TL;DR**
+
+Risk analysis is the process of identifying and analyzing potential issues that could negatively impact key business initiatives or projects. This process is done in order to help organizations avoid or mitigate those risks.
+
+## What Is Risk Analysis
+
+Risk analysis is the process of analyzing, determining, and defining the risk of danger to government agencies, and businesses. We also use it to determine risk to individuals, towns, cities, and whole national economies.
+
+Specifically, the analysis looks at potential natural and anthropogenic events. Anthropogenic events are events that human activity causes.
+
+In IT, we commonly use a risk analysis report to align technological goals with those of the company. IT stands for information technology. Risk analysis is both quantitative or qualitative.
+
+![Process](/files/risk_analysis.jpg)
+[Wallstreetmojo.com](https://www.wallstreetmojo.com/risk-analysis/)
+
+## Why You Might Want the Risk Analysis
+1. Completion of preventative risk analysis have potential to avoid making huge losses
+2. Risk analysis will help the business avoiding security breaches
+3. New policies can be drafted to prevent potential risks from eventuating
```

Comment:
You’ll be able to draft up new policies to prevent potential risks from eventuating. Changing active to passive is not rewriting.  (https://www.spews.org/marketing/why-risk-analysis-is-important/). This comment is regarding all 4 points.